environments/dbsql-nyc-taxi-trip-analysis/query_total_trips.tf

```diff
@@ -0,0 +1,65 @@
+resource "databricks_sql_query" "total_trips" {
+  data_source_id = databricks_sql_endpoint.this.data_source_id
+
+  name = "Total Trips"
```

Comment:
imagine that this example would be applied by 50 users in the same workspace (because people can just use terraform). please pick a prefix to all resources, for example `alphanumeric` or `user_name` from `databricks_current_user`